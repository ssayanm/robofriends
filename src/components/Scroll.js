import React from 'react';

const Scroll = (props) => {
	return (
	    <div className='tc' style={{overflowY: 'scroll', border: '1px solid #000', height: '800px'}}>
	    	{props.children}
	    </div>
  );
}

export default Scroll;
