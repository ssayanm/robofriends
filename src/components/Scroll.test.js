import { shallow } from "enzyme";
import React from "react";
import Scroll from "./Scroll";

it("expect to render Scroll component", () => {
  const scrollComponent = shallow(<Scroll />);
  expect(scrollComponent.debug()).toMatchSnapshot();
});
