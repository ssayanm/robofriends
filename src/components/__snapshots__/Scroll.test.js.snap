// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Scroll component 1`] = `"<div className=\\"tc\\" style={{...}} />"`;
